test/unit/lib/actions/sendEvent/createSendEvent.spec.js

```diff
@@ -72,6 +77,9 @@ describe("Send Event", () => {
       expect(decisionsCallbackStorage.triggerEvent).toHaveBeenCalledWith({
         decisions
       });
+      expect(sendEventCallbackStorage.triggerEvent).toHaveBeenCalledWith({
+        decisions
+      });
```

Comment:
Let's improve this assertion. In the real world, `triggerEvent` will now be called with an object containing both `decisions` and `propositions`, and we want to make sure that `sendEventCallbackStorage.triggerEvent` has been called with an object containing both `decisions` and `propositions` and that `decisionsCallbackStorage.triggerEvent` has been called with an object just containing `decisions`. You'll need to update our mock instance [here](https://github.com/adobe/reactor-extension-alloy/pull/160/commits/fcf907508ae696de24dfef217304876fcf2d7721#diff-17378a7b78223802022d653d7ae99071e331af5f9520eb536e32ad6fda9eb2afR18-R20) so that it resolves with a promise that contains both `decisions` and `propositions`, like Alloy now behaves.

Reply:
good point! thank you